src/view/components/EditEntry/index.tsx

```diff
@@ -0,0 +1,204 @@
+import * as React from 'react';
+import { Content, List, View, ListItem, Text, Grid, Row, Col, H1, Button, Input, Item } from 'native-base';
+
+import { Farmer } from '../../../store/modules/farmer/types';
+import { Dairy as MilkEntry, PartialDairy } from '../../../store/modules/dairy/types';
+import { Route } from '../../navigation/navigator';
+
+import { MapStateToProps, MapDispatchToProps, connect } from 'react-redux';
+import navActions from '../../../store/modules/nav/actions';
+import { InjectedFabProps } from '../../hoc/PageComposer/FabPage/index';
+import Composer from '../../hoc/PageComposer/index';
+import { State } from '../../../store/types';
+
+import dairyActions from '../../../store/modules/dairy/actions';
+
+import Styles from './style';
+
+
+interface OwnPropsType {
+  farmer: Farmer;
+  milkEntry: MilkEntry;
+}
+
+interface DispatchPropsType {
+  navigate(route: Route): void;
+  goBack(): void;
+  updateDairy(newEntry: PartialDairy): void;
+}
+
+interface StorePropsType {
+}
+
+type NestedPropsType = StorePropsType & DispatchPropsType & OwnPropsType;
+
+/** EditEntry PropsType */
+type PropsType = InjectedFabProps & NestedPropsType;
+
+interface OwnStateType {
+  volume: string;
+  quality: string;
+  costPerUnit: string;
+}
+
+/**
+ * Button color
+ */
+type ButtonColor = 'PRIMARY' | 'INFO';
+
+/**
+ * Page for EditEntry
+ * @requires farmer
+ * @requires milkEntry
+ * 
+ * @example 
+ *          <EditEntry
+ *          />
+ */
+class EditEntry extends React.Component<PropsType, OwnStateType> {
+
+  constructor(props: PropsType) {
+    super(props);
+    this.state = {
+      volume: this.props.milkEntry.volume,
+      quality: this.props.milkEntry.quality,
+      costPerUnit: this.props.milkEntry.costPerUnit,
+    };
+  }
+
+  /** Create page buttons */
+  private renderCancelButton = () => this.renderButton('Cancel', 'INFO', this.onCancelPress);
+  private renderSaveButton = () => this.renderButton('Save', 'PRIMARY', this.onSavePress);
+
+  /** Handle pressing cancel button */
+  private onCancelPress = () => this.props.goBack();
+  
+  /** Handle pressing save button */
+  private onSavePress = () => {
+    let newEntry: PartialDairy = {
+      uuid: this.props.milkEntry.uuid,
+      volume: this.state.volume,
+      quality: this.state.quality,
+      costPerUnit: this.state.costPerUnit,
+    };
+    this.props.updateDairy(newEntry);
```

Comment:
I think we are going with the name `milk` for everything instead of `dairy`

Reply:
I think this should auto-refactor when we refactor module `dairy` name, no?